pylint/checkers/__init__.py

```diff
@@ -46,28 +46,38 @@
 
 """
 
+from typing import Counter, Dict, List, Tuple, Union
```

Comment:
`typing.Counter` was added in `3.6.1`. Thus to use it, we need to add a guard unfortunately.
https://docs.python.org/3/library/typing.html#typing.Counter

Reply:
I borrowed the solution from:
https://github.com/PyCQA/pylint/blob/40cc2ffd7887959157aaf469e09585ec2be7f528/pylint/checkers/strings.py#L51-L52
Let me know if this is correct!